-- 1. Failure:  (@test-expect-condition.R#44)  ---------------------------------
`object` did not throw the expected error.

-- 2. Failure:  (@test-expect-condition.R#45)  ---------------------------------
`object` threw an unexpected error.
Message: !
Class:   simpleError/error/condition
Backtrace:
 1. testthat::expect_error(fail("!"), NA)
 7. testthat::fail("!")

-- 3. Error:  (@test-expect-condition.R#47)  -----------------------------------
xxx
Backtrace:
 1. testthat::expect_error(fail("xxx"), "zzz")
 7. testthat::fail("xxx")

